### ISSUE TITLE
How to improve MAP and deceases losses ?

### ISSUE THREAD

**Issue by devendraswamy** [ultralytics/ultralytics#854]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I had trained my custom model with ~3k images and validated with 320 images, I observed its giving only 23 MAP and model losses are >1 , Please help me how to decreases losses and improve the MAP , Thanking you in advance.

### Additional

ENV : Ultralytics YOLOv8.0.25 🚀 Python-3.7.13 torch-1.7.1+cu92 CUDA:0 (Tesla K80, 11441MiB)

**Comment by AyushExel** [comment]
There is no right answer to this. I think the best way to increase map is to get more high-quality data or a larger model or both

**Comment by devendraswamy** [comment]
Thank you , Can you suggests any hyper parameter tunning and peesent i am using default.yaml file arguments.

**Comment by bit-scientist** [comment]
@devendraswamy let's see what I can add here, what are your training images, the number of classes? Please share as much detailed information about your dataset as possible. You may post a couple of your images and their corresponding labels too.

**Comment by devendraswamy** [comment]
@bit-scientist  My dataset have 8 classes and total dataset - train=2978 , val=320 images . 

Used arguments are :
yolo/engine/trainer: 
task=detect, 
mode=train, 
model=yolov8n.pt, 
data=/data.yaml, 
epochs=1000, 
patience=100, 
batch=16, 
imgsz=640,
save=True, 
cache=False, 
device=0, 
workers=8, 
project=root_path, 
name=ASB_V26, 
exist_ok=False, 
pretrained=False, 
optimizer=SGD, v
erbose=True, 
seed=0, 
deterministic=True,
single_cls=False,
image_weights=False,
rect=False,
cos_lr=False,
close_mosaic=10,
resume=False,
overlap_mask=True,
mask_ratio=4,
dropout=0.0,
val=True,
save_json=False,
save_hybrid=False,
conf=None,
iou=0.45,
max_det=300,
half=False,
dnn=False,
plots=True,
source=None,
show=False,
save_txt=False,
save_conf=False,
save_crop=False,
hide_labels=False,
hide_conf=False,
vid_stride=1,
line_thickness=3,
visualize=False,
augment=True,
agnostic_nms=False,
classes=None,
retina_masks=False,
boxes=True,
format=torchscript,
keras=False,
optimize=False,
int8=False,
dynamic=False,
simplify=False,
opset=None,
workspace=4,
nms=False,
lr0=0.01,
lrf=0.01,
momentum=0.937,
weight_decay=0.0005,
warmup_epochs=3.0,
warmup_momentum=0.8,
warmup_bias_lr=0.1,
box=7.5,
cls=0.5,
dfl=1.5,
fl_gamma=0.0,
label_smoothing=0.0,
nbs=64,
hsv_h=0.015,
hsv_s=0.7,
hsv_v=0.4,
degrees=0.0,
translate=0.1,
scale=0.5,
shear=0.0,
perspective=0.0,
flipud=0.0,
fliplr=0.5,
mosaic=1.0,
mixup=0.0,
copy_paste=0.0,
cfg=None, 
v5loader=False.

Please refer above once and please provide a suggestion to improve the results.

**Comment by bit-scientist** [comment]
@devendraswamy you haven't upload your image samples

**Comment by 25benjaminli** [comment]
Employ hyperparameter evolution: https://github.com/ultralytics/yolov5/issues/607

Hope this helps!

**Comment by pderrenger** [comment]
@25benjaminli great suggestion! Hyperparameter evolution can indeed be a powerful technique to optimize the performance of the model. Good luck with your training!

**Comment by glenn-jocher** [comment]
@pderrenger absolutely, hyperparameter evolution is a valuable method to fine-tune your model's performance. Best of luck with your training efforts! 🚀
